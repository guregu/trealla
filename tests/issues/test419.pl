:- use_module(library(dif)).
:- initialization(main).

main :-
	portray_clause(c:t((dif(A,B),A=[]*C,B=[[]|D]),(A=[]*C,B=[[]|D],dif(A,B)))).
