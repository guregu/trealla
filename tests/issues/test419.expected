c:t((dif(A,B),A=[]*C,B=[[]|D]),(A=[]*C,B=[[]|D],dif(A,B))).
